app/Policies/JobPolicy.php

```diff
@@ -113,25 +111,44 @@ public function submitForReview(User $user, JobPoster $jobPoster)
      */
     public function reviewApplicationsFor(User $user, JobPoster $jobPoster)
     {
-        // Only managers can review applications, and only for their own jobs.
-        return $user->isManager() &&
-            $jobPoster->manager->user->id == $user->id &&
-            $jobPoster->isClosed();
+        // Managers can only review applications their own jobs.
+        // HR Advisors can review applications for jobs they manage.
+        // The job must always be closed.
+        $authManager = $user->isManager() && $jobPoster->manager->user->id == $user->id;
+        $authHr = $user->isHrAdvisor() && $this->manage($user, $jobPoster);
+
+        return $jobPoster->isClosed() && ($authManager || $authHr);
     }
 
-     /**
+    /**
+     * Determin whether the user is an HR Advisor with permission to manage this job.
+     *
+     * @param User $user
+     * @param JobPoster $jobPoster
+     * @return void
+     */
+    public function manage(User $user, JobPoster $jobPoster)
+    {
+        return ($user->isManager() &&
+            $jobPoster->manager->user->id == $user->id) ||
+            ($user->isHrAdvisor()
+                && $this->view($user, $jobPoster)
+                && $user->hr_advisor->claimed_job_ids->includes($jobPoster->id));
```

Comment:
I got an error for this. I don't know `includes()` exists anymore. I switched it to `in_array()`.

Reply:
Ah, you're right. I'm trying to use laravel collection methods - it should be [contains](https://laravel.com/docs/5.2/collections#method-contains).